data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -891,7 +909,6 @@
     "type": "MONSTER",
     "id": "mon_apatosaurus",
     "name": { "str_sp": "apatosaurus" },
-    "looks_like": "mon_camptosaurus",
```

Comment:
Just checking, is this intentional?

Reply:
Yep, it was on the list waiting for the next PR, but this became the next PR